pkg/machine/ignition/ignition.go

```diff
@@ -743,6 +776,37 @@ func (i *IgnitionBuilder) Build() error {
 	return i.dynamicIgnition.Write()
 }
 
+func GetNetRecoveryFile() string {
+	return `#!/bin/bash
+# Verify network health, and bounce the network device if host connectivity
+# is lost. This is a temporary workaround for a known rare qemu/virtio issue
+# that affects some systems
+
+sleep 120 # allow time for network setup on initial boot
+while true; do
+  sleep 30
+  curl -s -o /dev/null --connect-timeout 10 http://192.168.127.1/health
+  if [ "$?" != "0" ]; then
+    echo "bouncing nic due to loss of connectivity with host"
```

Comment:
where does this line is reported to ? to see if it occurred or not in my podman machine

Reply:
The unit file sends stdout&stderr to the system journal, so can be found using journalctl.